Randomly selected response on error.

dude22072 opened the pull request:
The discord chat asked for it, so here it is.

dude22072 committed:
Randomly selected response on error.

The discord chat asked for it, so here it is.

dude22072 committed:
Moving random to util/index

dude22072 committed:
Moving random to util/index - Part 2

arya-s commented on src/commands/query.js:
```diff
@@ -2,6 +2,7 @@ import config from '../../config';
 import { createClient } from 'wolfram-alpha';
 
 const wolfram = createClient(config.wolframToken);
+var errorResponses = ['I couldn\'t find anything.', 'Yuka-chan is smarter than me, she would know.', 'Only the Fox God knows.'];
```
Break this into multiple limes.

arya-s commented on src/commands/query.js:
```diff
@@ -23,10 +24,9 @@ class Command {
       if (error || result.length === 0) {
-
-        payload.message.reply('I couldn\'t find anything.');
+        var rand = getRandom(0,3);
```
You need to import getRandom.

Also remove the variable and plug it directly into the array on the next line.

arya-s commented on src/util/index.js:
```diff
@@ -34,4 +34,11 @@ export const getImage = function (message) {
+};
+
+/*
+ * Get a random number bounded by min(inclusive) and max(exclusive)
```
Please add proper jsdoc (see other methods)

dude22072 committed:
jsdoc for getRandom()

dude22072 committed:
Formatting of query.js

errorResponses on multiple lines
getRandom in the reply instead of it's own variable

arya-s commented on src/commands/query.js:
```diff
@@ -1,7 +1,11 @@
+import { getRandom } from '../util';
 
 const wolfram = createClient(config.wolframToken);
+var errorResponses = ['I couldn\'t find anything.',
```
Should be const instead of var.

dude22072 committed:
Const, not var. /kappa
